stripe/src/main/java/com/stripe/android/AlipayAuthenticationHandler.kt

```diff
@@ -0,0 +1,24 @@
+package com.stripe.android
+
+import androidx.annotation.WorkerThread
+import com.stripe.android.model.PaymentIntent
+
+/**
+ * Interface for handling [PaymentIntent] authentication with the Alipay SDK
+ * @see <a href="https://intl.alipay.com/docs/ac/app/sdk_integration">Alipay Documentation</a>
+ *
+ * To authenticate using the Alipay SDK, pass the data String to PayTask#payV2
+ * <pre>
+ * new AlipayAuthenticationHandler() {
+ *  @NotNull
+ *  @Override
+ *  Map<String, String> authenticate(@NotNull String data) {
+ *      return new PayTask(this).payV2(data, true)
+ *  }
+ * }
+ * </pre>
+ */
+interface AlipayAuthenticationHandler {
```

Comment:
Any opinions on naming here?

Reply:
maybe `onAuthenticate` or `onAuthenticationRequest`?

Reply:
Also probably not `Handler`, since that's overloaded in Android